web/src/lib/components/shared-components/upload-cover.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import { fade } from 'svelte/transition';
```

Comment:
Let's change the name of the file to `drag-and-drop-upload-overlay` for clarity

Reply:
Changed.